update.js

```diff
@@ -0,0 +1,51 @@
+const files = require("./lib/files")
+const ccdDef = require('./lib/modify-ccd-definition')
+const cmdLine = require('./lib/command-line-parse')
+const log = (out) => console.log(out)
+
+const start = async () => {
+    var cmdline = new cmdLine.ParsedCmdLine()
+    if (!cmdline.valid) {
+        log("invalid command line")
+        return
+    }
+
+    log("Import...\n loading workbook: " + cmdline.filename);
```

Comment:
Use the same logging approach in both JS files. Here we have `\n` separator and single line. In other file we have two separate calls to `console.log`